templates/bundles/DemosPlanCoreBundle/DemosPlanProcedure/administration_edit.html.twig

```diff
@@ -642,7 +573,10 @@
                                     } %}
                                     <dp-date-range-picker
                                         class="u-2-of-4"
-                                        data-cy="procedureSettingsParticipationDate"
+                                        :data-cy="{
```

Comment:
Shouldn't this prop be a string?

Reply:
Means. It has to be changed with the next demosplan-ui release. But for now it has to be an object. The core-PR that changed this got merged before the new demosplan-ui was released